.vscode/tasks.json

```diff
@@ -1,22 +1,20 @@
 {
-	"version": "2.0.0",
-	"tasks": [
-		{
-			"type": "npm",
-			"script": "lint",
-			"problemMatcher": [],
-			"label": "npm: lint",
-			"detail": "eslint ./src"
-		},
-		{
-			"type": "typescript",
-			"tsconfig": "tsconfig.json",
-			"option": "watch",
-			"problemMatcher": [
-				"$tsc-watch"
-			],
-			"group": "build",
-			"label": "tsc: watch - tsconfig.json"
-		}
-	]
-}
\ No newline at end of file
+  "version": "2.0.0",
```

Comment:
Is this file and extensions file necessary?

Reply:
Some automatic linting has snuck in, I'll remove in a sec